surveys/views.py

```diff
@@ -405,7 +405,7 @@ class SurveyListEdit(ListView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context['surveys'] = context['surveys'].exclude(active=False).filter(published=False).order_by('-updated')
```

Comment:
Making sure templates don't show up as editable or runnable surveys!

Reply:
Nice catch! This line is getting a little long (117 characters), could we add in a linebreak? Maybe something like:

```python
context['surveys'] = context['surveys'].filter(
    active=True, is_cloneable=False, published=False
).order_by('-updated')
```